client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -4,14 +4,19 @@
 import React, { Component, PropTypes } from 'react';
 import { localize } from 'i18n-calypso';
 import { connect } from 'react-redux';
+import get from 'lodash/get';
+import assign from 'lodash/assign';
```

Comment:
We are currently using the `import { assign } from 'lodash'` form for importing lodash methods. You can combine `{ get, assign }` in a single statement.